no_drama/__main__.py

```diff
@@ -0,0 +1,179 @@
+
+    archive_basename = "%s_%s" % (cli_args.name, cli_args.label)
+    archive_name = shutil.make_archive(archive_basename,
+                                       'zip',
+                                       root_dir=staging_dir,
+                                       base_dir=cli_args.label)
+
+    print("generated build at %s" % archive_name)
+    shutil.rmtree(staging_dir)
+    print("deleted temporary files")
+
+
+def inject_configuration(cli_args):
+    staging_dir = tempfile.mkdtemp()
+    build_zip_path = os.path.join(staging_dir, 'build.zip')
+
+    shutil.copyfile(cli_args.build_zip, build_zip_path)
+
+    build_zip = zipfile.ZipFile(build_zip_path, mode="a")
+
+    # figuring out where things are in this zip file...
+    all_the_wheels = fnmatch.filter(build_zip.namelist(), '*/wheels/*.whl')
+    zip_wheel_dir = os.path.dirname(all_the_wheels[0])
+    zip_root = os.path.normpath(os.path.join(zip_wheel_dir,'../'))
+
+    # move just the wheels we want into the bundle dir
+    wheel_destination = os.path.join(staging_dir, 'wheels')
+    if cli_args.requirements_file:
+        save_wheels(requirements_file=cli_args.requirements_file,
+                    destination=wheel_destination)
+        wheel_pattern = os.path.join(staging_dir, 'wheels/*.whl')
+
+        saved_wheels = glob.glob(wheel_pattern)
+
+        for wheel_path in saved_wheels:
+            name = os.path.join(zip_wheel_dir, os.path.basename(wheel_path))
+            build_zip.write(wheel_path, arcname=name)
+
+    build_zip.write(cli_args.vars, os.path.join(zip_root, 'environment.json'))
+
+    if cli_args.paths:
+        build_zip.write(cli_args.paths, os.path.join(zip_root, 'paths.json'))
+
+    if cli_args.append_wsgi:
+        build_zip.write(cli_args.append_wsgi,
+                        os.path.join(zip_root, 'extended.wsgi'))
+
+    build_zip.close()
+    build_filename = os.path.basename(cli_args.build_zip)
+    name, ext = os.path.splitext(build_filename)
+    release_filename = "%s_%s%s" % (name, cli_args.slug, ext)
+    shutil.copyfile(build_zip_path, release_filename)
+
+
+def main():
+    parser = argparse.ArgumentParser(description="drama-free deployable django"
+                                     " projects")
+
+    subparsers = parser.add_subparsers()
+    build_parser = subparsers.add_parser('build')
+    release_parser = subparsers.add_parser('release')
+
+    # build parser arguments
+    build_parser.add_argument('project_path')
+    build_parser.add_argument('requirements_file', help="just like you would"
+                              " 'pip install -r'")
+
+    build_parser.add_argument('name', help="name of this project")
+
+    build_parser.add_argument('label', help="a label for this build-- "
+                        "maybe a build ID or version number")
+
+    build_parser.set_defaults(func=stage_bundle)
+
+    # release parser arguments
+    release_parser.add_argument('build_zip', help='path to a zip file'
```

Comment:
You could create a dictionary and iterate it and call add arguments in a loop, reducing the lines of code.